Ask users to tell us when they've made a proposal (#2584)
alphagov/govuk-design-system: src/community/propose-a-component-or-pattern/index.md

## 2. Raise an issue

> If your idea is not on the list, [raise an issue](https://github.com/alphagov/govuk-design-system-backlog/issues/new). Once you do, [let us know](/get-in-touch/) so a member of the Design System team can review your proposal.

Author: ```suggestion
If your idea is not on the list, [raise an issue](https://github.com/alphagov/govuk-design-system-backlog/issues/new). Once you do, [let us know](/get-in-touch/) a member of the Design System team will review your proposal and add it to the list.
```
